aljuarismi/dimensionality.py

```diff
@@ -7,29 +7,118 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import click
 import khiva as kv
 
 import aljuarismi as al
 
 
-def reduce_datapoints(timeseries, parameters):
+def get_int_number(parameters):
     """
-
-    :param timeseries:
-    :param num_points:
-    :return:
+    Obtains or asks for a integer number.
+    :param parameters: the parameters of the function(number,...).
+    :return: An integer number.
     """
-    if timeseries.columns.size > 1:
-        timeseries = timeseries[al.obtain_column(timeseries)]
-    if parameters.get("number") == '':
+    if parameters["number"]:
+        num_points = int(parameters["number"])
+    else:
         print('How many point do you want to reduce it?')
```

Comment:
What should be the value of epsilon?

Reply:
For obtaining the epsilon value, it has his own function which is called get_epsilon, before called get_number or get_float_number